### DISCUSSION TITLE
Kupo re-synchronization since some slots

### DISCUSSION THREAD

**Discussion by Timekiller7** [CardanoSolutions/kupo#144] [General]
We started synchronizing Kupo since slot `n`. 
Can we then get Kupo synchronized from some another slot - synchronizing only those parts that were before slot `n` without synchronizing slots after?

**Comment by KtorZ** [comment]
No. 

It can synchronize from an earlier point but it then have to re-synchronize blocks after `n` for data-consistency reasons as new elements from earlier blocks could change the view of the system after `n`.

**Reply by Timekiller7** [reply]
Thanks a lot!
